app/src/widgets/todoist/properties.ts

```diff
@@ -0,0 +1,12 @@
+import WidgetCategory from "../categories";
+
+export const widgetType = "todoist";
+export const category = WidgetCategory.Productivity;
+export const initialHeight = 2;
+export const initialWidth = 3;
+export const initialOptions = {
+  token: "xxxxx"
```

Comment:
The `token` should be empty or undefined by default. That way, you can use `<WidgetUnconfigured>` as long as it is missing (see the usage in other widgets).